src/ripple/test/jtx/impl/Env_test.cpp

```diff
@@ -561,6 +561,21 @@ class Env_test : public beast::unit_test::suite
                 );
     }
 
+    void testSig()
+    {
+        using namespace jtx;
+        Env env(*this);
+
+        env.fund(XRP(10000), "alice");
+        auto const baseFee = env.app().config().FEE_DEFAULT;
+        std::uint32_t const aliceSeq = env.seq ("alice");
+        
+        Json::Value jsonNoop = env.json (
+            noop ("alice"), fee(baseFee), seq(aliceSeq), sig("alice"));
+        JTx jt = env.jt (jsonNoop);
+        env (jt);
```

Comment:
@scottschurr Sounds like Env_test.cpp is the right place then, you justification is sound